src/components/app/App.js

```diff
@@ -20,7 +20,6 @@ function App() {
         <Footer />
       {/*  todo: здесь нужно использовать UserCard по аналогии с Header, как пропсу user передать usersList[0]*/}
         <UserCard user={usersList[0]}/>
```

Comment:
if you want to know better what is destructuring you can google
'how to destructure only first item from array'
and apply this here
        <UserCard user={usersList[0]}/>
(to avoid writting this [0])

again, not required, but good to know